Sources/HttpServer/Route/Route.swift

```diff
@@ -3,9 +3,24 @@ public class Route {
     public var method: HttpMethod
     public var action: HttpAction
 
+//    let routeMap: [String: ()] = [:]
+
     public init(url: String, method: HttpMethod, action: HttpAction) {
         self.url = url
         self.method = method
         self.action = action
     }
+//
+//    func post(){
+//
+//    }
+//
+//    func put() {
+//        returns response
+//    }
+//
+//
+//    func options() {
+//
+//    }
```

Comment:
What are all of these comments doing here? Get rid of these!

Reply:
Sorry. I was brainstorming and then forgot about these.